.github/ISSUE_TEMPLATE/7_cherry_pick.md

```diff
@@ -0,0 +1,24 @@
+---
+name: Request a cherry-pick.
+about: As a contributor, you would like to request that a feature be cherry-picked into a release.
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+commit(s) to merge: (Enter the shortened commit hash of the commit that has been merged to master/main. <fabfeed>)
```

Comment:
I was going to say it's probably easier to link to the GitHub URL for the PR, but it's a nice captcha for whether people should be filing these.  Not to gatekeep, but if you don't know what a commit sha is you probably won't be able to fill out the rest of this successfully.

Reply:
Agreed.